packages/veraswap-sdk/src/calls/getTransferRemoteWithKernelCalls.ts

```diff
@@ -12,9 +12,12 @@ import { Execute } from "../artifacts/Execute.js";
 import { getExecMode } from "@zerodev/sdk";
 import { CALL_TYPE, EXEC_TYPE } from "@zerodev/sdk/constants";
 import { getOwnableExecutorExecuteCalls } from "./getOwnableExecutorExecuteCalls.js";
+import { TokenStandard } from "../types/Token.js";
+import { getOrbiterETHTransferTransaction } from "../orbiter/getOrbiterETHTransferTransaction.js";
+import { OrbiterParams } from "../types/OrbiterParams.js";
 
 export interface GetTransferRemoteWithKernelCallsParams extends GetCallsParams {
-    tokenStandard: "HypERC20" | "HypERC20Collateral";
+    tokenStandard: TokenStandard;
```

Comment:
Why not have the type be more constrained to `HypERC20 | HypERC20Collateral | NativeToken` this makes it clearer that these are the supported token types. We can still keep the runtime check as well.